plaidml/bridge/openvino/ops/roi_pooling.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "plaidml/op/op.h"
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+#include <tuple>
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace {
+
+template <typename T>
+std::vector<T> cast_constant_operand(size_t operand_idx, ngraph::Node* layer) {
+  auto* ngraph_const = ngraph::as_type<ngraph::op::Constant>(layer->get_input_node_ptr(operand_idx));
+  if (ngraph_const) {
+    return ngraph_const->cast_vector<T>();
+  } else {
+    THROW_IE_EXCEPTION
+        << "Dynamic coordinate is not currently supported by PlaidML plugin; all coordinate must be Constants. ";
+  }
+}
+
+edsl::Tensor crop_max_pooling(edsl::Tensor I, std::vector<float>& coord, int64_t pooled_h, int64_t pooled_w) {
```

Comment:
Looks like `coord` should be `const std::vector<float>& coord`? Is this intended to be an in/out parameter?

Reply:
`coord` don't intended to be an in/out parameter, so you are right, it should be  `const std::vector<float>& coord`. I fix this in PR https://github.com/plaidml/plaidml/pull/1641